content/_index.md

```diff
@@ -49,7 +49,10 @@ You may want to add language similar to this to introduce your code of conduct:
 > Conduct. By participating in this project you agree to abide by its
 > terms.
 
-You may also use the permalinks given above to reference from your project home page.
+You may also use the permalinks given above to reference from your project home page. If you are using a markdown README file, you may want to add a badge like this one ![Contributor Covenant Badge](https://img.shields.io/badge/Contributor%20Covenant-v1.4%20adopted-ff69b4.svg) using the code below.
```

Comment:
I'd prefer that this get moved down one paragraph, since the reporting contact email note is really important. Otherwise this looks great!

Reply:
ok. done.